src/organizations/interfaces/organization-domain.interface.ts

```diff
@@ -2,4 +2,7 @@ export interface OrganizationDomain {
   object: 'organization_domain';
   id: string;
   domain: string;
+  verification_token?: string;
+  verification_stragegy?: 'dns' | 'manual';
```

Comment:
typo here, but also i am wondering if we want to expose these properties in the SDK? @oliverzheng what do you think?

Reply:
we can get into trouble when using enums / string literals since then adding a new value is a breaking API change. so perhaps we should set these to string types?

Reply:
Yeah we can either make it a string type, or a `'dns' | 'manual' | 'unknown'` and have `'unknown'` be a special value that deserialization uses for any value, well, unknown to the existing values. `string` is probably simple enough here though.

Reply:
sounds like you're ok w/ exposing these 3 new properties?

Reply:
nvm i'm going to remove these for now as they aren't deserialized properly.